oci/spec_opts.go

```diff
@@ -333,7 +333,13 @@ func WithImageConfigArgs(image Image, args []string) SpecOpts {
 			return WithAdditionalGIDs("root")(ctx, client, c, s)
 		} else if s.Windows != nil {
 			s.Process.Env = config.Env
```

Comment:
Should s.Process.Env be appended to ^ as done for Linux in L313?

Reply:
Hard to answer for me actually, as I have arguments "for" and "against". So on one hand it was like this before but on other hand why do we have different behavior for linux and windows? Also why do we have different "merge"/"apply" behaviors, so all other parameters we replace/override, but for env we use "append" behavior, what are use cases to have this exception?

Reply:
@jterry75 any thoughts on the above?